src/pipx/package_specifier.py

```diff
@@ -161,7 +161,22 @@ def parse_specifier_for_install(package_spec: str, pip_args: List[str]) -> Tuple
         )
         pip_args.remove("--editable")
 
-    return (package_or_url, pip_args)
+    pip_args = [option for pip_arg in pip_args for option in pip_arg.split(" ")]
+
+    for option in pip_args:
+        if "-c" in option or "--constraint" in option:
+            option_list = option.split("=")
```

Comment:
This won't work when the constraints file is given without the equals sign.

Reply:
changed